srtcore/filelist_win32_shared.maf

```diff
@@ -0,0 +1,7 @@
+
+
+SOURCES
+srt_shared.rc
+
+PRIVATE HEADERS
+../version.h
```

Comment:
I don't like this extra maf file. Might be better to add additional sections to the existing `filelist.maf` like
```
SOURCES_WIN32_SHARED
srt_shared.rc

PRIVATE_HEADERS_WIN32_SHARED
../version.h
```
and then read those section with `
```
MafReadDir(srtcore filelist.maf 
	SOURCES_WIN32_SHARED SOURCES_srt_shared_win32 
	PRIVATE_HEADERS_WIN32_SHARED HEADERS_srt_shared_win32 
)
```

Also might be worth adding a new line?

Reply:
Happy to do that - was just my natural aversion to messing with how things were before. Please check i did not misunderstand you.